### PR TITLE
Added rails example of importing data from google spreadsheet in google charts

### PR DESCRIPTION
**Pull request by Prakriti-nith**

### CODE REVIEW

**Review by Shekharrajak — CHANGES_REQUESTED** [review]
There is already `demo_rails` folder. Examples are expected to be there in new controller and page. Real world examples will be appreciated.

**Comment by Prakriti-nith** [comment]
I have added this in the application_controller as it was a single example.

File: demo_rails/app/views/application/googlecharts.erb [hunk 1]
```diff
@@ -47,6 +47,17 @@ charts is not coming in 3d, since google_visualr loading only corechart package
 </p>
 <br>
 
+<h3> Importing Data from Google spreadsheet </h3>
+<p>
```
Comment (Shekharrajak) [inline demo_rails/app/views/application/googlecharts.erb]: It is better you show table and URL ,then display chart. I think we must show the simple code as well how it is working.

Comment (Prakriti-nith) [reply demo_rails/app/views/application/googlecharts.erb]: I have added the URL, the table and code to show what js is generated. I have also added another example with easy data.

**Comment by Shekharrajak** [comment]
Any update in this PR? Screenshot or GIF ?

**Comment by Prakriti-nith** [comment]
gs1
gs2

**Review by Shekharrajak — APPROVED** [review]

**Pull request merged**